homeassistant/components/homematic/entity.py

```diff
@@ -268,6 +277,33 @@ def _update_variables(self, now):
         if state_change:
             self.schedule_update_ha_state()
 
+    def _update_duty_cycle(self, now):
+        """Retrieve duty cycle."""
+        if "DutyCycle" in self._variables:
+            # e.g. RaspberryMatic already sets DutyCycle system variable
+            self._duty_cycle = None
+            return
+        bidcos_interfaces = self._homematic.listBidcosInterfaces(self._name)
+        if bidcos_interfaces is None:
+            return
+        _LOGGER.debug("bidcos_interfaces: %s", bidcos_interfaces)
+        if len(bidcos_interfaces) == 0:
+            return
+        if len(bidcos_interfaces) > 1:
+            _LOGGER.warning(
```

Comment:
This (and the warning below) will result in constant warning-spam in case the conditions for getting the duty cycle isn't matched.  I get why you chose to make this a warning. But I would vote to either change this to `info` or `debug` so people can filter this out more easily, or don't even log it at all. If there's nothing that can be done to get the DC, those messages might as well be hidden all together. 🤷‍♂️